python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -46,6 +55,12 @@ class DeclarativeSubmetricSchema(SubmetricSchema):
     def __init__(self, resolvers: List[ResolverSpec], default_config: Optional[MetricConfig] = None) -> None:
         self._default_config = default_config or MetricConfig()
         self._resolvers = resolvers.copy()
+        for resolver in resolvers:
+            for metric_spec in resolver.metrics:
+                if issubclass(metric_spec.metric, MultiMetric):  # and not resolver.exclude:
```

Comment:
This should prevent future multi metric deserialization problems caused by nested multi metrics.